src/modules/categories/components/CategoryListItem.js

```diff
@@ -3,14 +3,14 @@
 import * as React from 'react'
 
 import iconPlaceholder from '../assets/IconPlaceholder.png'
-import styled from 'styled-components/native'
-import type { ThemeType } from 'modules/theme/constants/theme'
+import styled, { type StyledComponent } from 'styled-components/native'
+import type { ThemeType } from '../../../modules/theme/constants/theme'
 import FastImage from 'react-native-fast-image'
 import CategoryCaption from './CategoryCaption'
 import StyledLink from './StyledLink'
 import SubCategoryListItem from './SubCategoryListItem'
 
-const Row = styled.View`
+const Row: StyledComponent<{}, {}, *> = styled.View`
```

Comment:
I think you should also use the `ThemeType` as second argument here.

Reply:
Same for some other occurances.

Reply:
As those styled components do not require a Theme I decided to use `{}`